ci/build_python.sh

```diff
@@ -26,4 +26,21 @@ rapids-mamba-retry mambabuild \
   --channel "${RAPIDS_CONDA_BLD_OUTPUT_DIR}" \
   conda/recipes/raft-dask
 
+# Build ann-bench for each cuda and python version
+rapids-mamba-retry mambabuild \
+--no-test \
+--channel "${CPP_CHANNEL}" \
+--channel "${RAPIDS_CONDA_BLD_OUTPUT_DIR}" \
+conda/recipes/raft-ann-bench
+
+# Build ann-bench-cpu only in CUDA 12 jobs since it only depends on python
+# version
+if [[ ${CUDA_VERSION} == "11.8.0" ]]; then
```

Comment:
The comment does not match the conditional. Also, I think the variable name that our CI containers use is `RAPIDS_CUDA_VERSION`. Can you verify if `CUDA_VERSION` is set?

Reply:
yes, CUDA_VERSION is set, you can see it in the logs https://github.com/rapidsai/raft/actions/runs/6051700515/job/16424916331#step:7:54 which have both with and without `RAPIDS_` prefix

Reply:
Better yet, copy this snippet (but use CUDA 12 to match the conditional as @divyegala noted above):

```suggestion
# Build ann-bench-cpu only in CUDA 12 jobs since it only depends on python
# version
RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
if [[ ${RAPIDS_CUDA_MAJOR} == "12" ]]; then
```

https://github.com/rapidsai/cugraph/blob/2b4118aee4af912d74ce1ebe7adc39cf596899ef/ci/build_python.sh#L49-L51

Reply:
we require to build in the CUDA 11 jobs for now, but otherwise will use it!